test/local/senders/index.js

```diff
@@ -92,7 +92,7 @@ describe('lib/senders/index', () => {
             return email.sendVerifyLoginEmail(EMAILS, acct, {code: code})
           })
           .then(() => {
-            assert.equal(bounces.check.callCount, 3)
+            assert.equal(bounces.check.callCount, 2)
```

Comment:
The call counts changed because we no longer check bounciness of unverified secondary emails.